src/System.Runtime.Numerics/tests/BigInteger/Comparison.cs

```diff
@@ -359,12 +359,25 @@ private static void RunNegativeTests(Random random)
             Assert.Equal(false, BigInteger.Zero.Equals((Object)"0"));
         }
 
-        [Fact]
-        public static void IComparable_Invalid()
+        public static void IComparable_Invalid(string paramName)
         {
             IComparable comparable = new BigInteger();
             Assert.Equal(1, comparable.CompareTo(null));
-            Assert.Throws<ArgumentException>("obj", () => comparable.CompareTo(0)); // Obj is not a BigInteger
+            Assert.Throws<ArgumentException>(paramName, () => comparable.CompareTo(0)); // Obj is not a BigInteger
```

Comment:
It doesn't have to be. it depends on the behavior we desire. if netcore behavior is correct or better one then we can have differences. if it doesn't matter to have this netcore behavior then we can change it to match the desktop. from what I am seeing it make sense to keep netcore behavior